main_comment.go

```diff
@@ -89,6 +89,12 @@ func processGitHubComment(ctx *gin.Context, comment *github.IssueCommentEvent, g
 				log.Errorf("Could not start build: %s", err.Error())
 			}
 		}
+	case strings.Contains(body, commandCherryPickBranch):
+		log.Infof("Attempting to cherry-pick the changes in PR: %v", pr)
```

Comment:
Lol, I just wrote about the `-` in cherry-pick/cherry pick and I see that even bot itself is not consistent :P